benthomasson/ansible-events-ui
# Fix Read Activation with optional params bug

Fix a bug reading activations created with optional params.

Testing:

1. Create a couple of activations using the following request bodies:

a. 
```
{
    "name": "demo",
    "rulebook_id": 1,
    "inventory_id": 1,
    "extra_var_id": 1
}
```

b. 
```
{
    "name": "demo",
    "rulebook_id": 1,
    "inventory_id": 1
}
```

2. Retrieve all activations calling GET on `http://localhost:8080/api/activations`.
3. Verify that you can see all existing activations including the ones with and without `extra_var`.

Resolves: AAP-6090
